ffi/jpeg.lua

```diff
@@ -90,4 +87,8 @@ function Jpeg.encodeToFile(filename, source_ptr, w, h, quality, color_type, subs
     end
 end
 
-return Jpeg
\ No newline at end of file
+function Jpeg.writeBMP(filename, source_ptr, w, h)
+    turbojpeg.tjSaveImage(ffi.cast("const char *", filename), source_ptr, w, 0, h, turbojpeg.TJPF_BGR, 2)
```

Comment:
You *might* be able to pass filename directly and let LuaJIT sort it out (I *think* that works).

Reply:
Not *quite* sure about the pixelformat either, from the docs, I'm not sure it matters if it's not gray or cmyk, but, in any case, that's the format of the *input* buffer: and ours is RGB, not BGR ;).

Reply:
> You _might_ be able to pass filename directly and let LuaJIT sort it out (I _think_ that works).

Yes, that works.

Reply:
> and ours is RGB, not BGR ;).

Right, this is a leftover. The fileformat stays netbmp.